lightcurve/static/lightcurve/js/lightcurve.js

```diff
@@ -74,18 +73,20 @@ $(function() {
     return false;
   });
 
-  $(document).on('click', '.edit-name', function(e) {
-    e.preventDefault();
+  $('.js-edit-name').on('click', function() {
     var filename = $(this).prev(".lightcurve-name").text();
     $( $(this).prev(".lightcurve-name") ).replaceWith( "<input type='name' class='input_name' name='input_name' value='" + filename + "'>" );
```

Comment:
You should do display:none on this input too and simply set the value :)

Reply:
Done.